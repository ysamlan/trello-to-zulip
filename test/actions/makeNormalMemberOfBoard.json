{
  "data": {
    "board": {
      "id": "b1idb1idb1idb1idb1idb1id",
      "name": "Board One"
    },
    "idMember": "51b9bca64517bd073d0031f5"
  },
  "date": "2013-11-03T20:23:46.185Z",
  "id": "actidactidactidactidacti",
  "idMemberCreator": "mcidmcidmcidmcidmcidmcid",
  "member": {
    "avatarHash": "mahmahmahmahmahmahmahmahmahmahma",
    "fullName": "Member Full Name",
    "id": "midmidmidmidmidmidmidmid",
    "initials": "MFN",
    "username": "mfname"
  },
  "memberCreator": {
    "avatarHash": "mcahmcahmcahmcahmcahmcahmcahmcah",
    "fullName": "Member Creator Full Name",
    "id": "mcidmcidmcidmcidmcidmcid",
    "initials": "MCFN",
    "username": "mcfname"
  },
  "type": "makeNormalMemberOfBoard"
}