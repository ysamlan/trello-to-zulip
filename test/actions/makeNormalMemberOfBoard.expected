Member Creator Full Name performed makeNormalMemberOfBoard on [Board One](https://trello.com/board/b1idb1idb1idb1idb1idb1id)